src/Html/BlockBuilder.php

```diff
@@ -37,6 +38,7 @@ public function startBlock($name)
      * Helper for endBlock and also clears the output buffer.
      * @param boolean $append Indicates that the new content should be appended to the existing block content.
      * @return void
+     * @throws Exception if there are no items in the block stack
```

Comment:
@jacobdekeizer could you make sure this and the following throws declarations are using the root namespace `\Exception`?
```suggestion
     * @throws \Exception if there are no items in the block stack
```